proposal (copy).md

```diff
@@ -0,0 +1,139 @@
+---
+title: Pandas
```

Comment:
Do we need a more expanded title?

Reply:
Thoughts on "Ensuring the continued growth of Pandas". I like *growth* of pandas, but maybe not for the title :)